# The linear velocity of the turtle in m/s
float32 linear

# The angular velocity of the turtle in rad/s
float32 angular